src/AddressBond.jsx

```diff
@@ -15,34 +14,35 @@ class AddressBond extends InputBond {
 			bonds.me
 		], (reg, me) => ({
 			registry: isNullData(reg) ? null : reg,
-			internal: n == 'null' ? '0x0000000000000000000000000000000000000000' : n == 'me' ? me : null
+			internal: n === 'null' ? '0x0000000000000000000000000000000000000000' : n === 'me' ? me : null
```

Comment:
No idea what this code does, maybe add some comments? Seems weird that n can be 'null' string.

Reply:
Gav wrote this (and most of this whole repo), so there are some snippets that are also new to me but I'll try to figure it out :)